express/src/lib/checkOtp.ts

```diff
@@ -0,0 +1,28 @@
+import {validationResult} from "./validationResult";
+
+export function validateOtp(otp: string): validationResult {
+    if (isEmpty(otp)) {
+        return {isValid: false, errorMessage: "Enter the 6 digit security code"};
```

Comment:
This class is a good pattern for validators.

Did we decide in the end we wanted have an `isValid` bool in addition to a nullable error message? (Either is fine)

Reply:
I've gone with it because it's a little more fluent than having validity implied by the nullity of a string - as tempting as using the falsy-ness of an undefined string is.